src/libraries/Common/tests/System/GenericMathHelpers.cs

```diff
@@ -188,11 +188,19 @@ public static class FloatingPointHelper<TSelf>
         public static int WriteSignificandLittleEndian(TSelf value, Span<byte> destination) => value.WriteSignificandLittleEndian(destination);
     }
 
-    public static class FloatingPointIeee754Helper<TSelf>
-        where TSelf : IFloatingPointIeee754<TSelf>
+    public static class FloatingPointConstantsHelper<TSelf>
```

Comment:
I'm sure it was discussed, but why did we go with the "Helper" suffix rather than just having the class be named "FloatingPointConstants<TSelf>"?